app/meetings/detail/route.ts

```diff
@@ -7,8 +7,8 @@ import Analytics from 'ember-osf-web/services/analytics';
 export default class MeetingsDetail extends Route {
     @service analytics!: Analytics;
 
-    model(params: string) {
-        return params;
+    model(params: Record<string, string>) {
+        return params.meeting_id;
```

Comment:
A pattern we've used is to have the model hook return an object that includes a task instance for loading the model (see https://github.com/CenterForOpenScience/ember-osf-web/blob/develop/app/settings/tokens/edit/route.ts for an example). This task instance can be passed to PaginatedList::HasMany as `@modelTaskInstance` and it will yield to it and show a loader while the task is running.